packages/aws-cdk/lib/init-templates/app/typescript/tsconfig.json

```diff
@@ -20,11 +21,11 @@
     "experimentalDecorators": true,
     "strictPropertyInitialization": false,
     "typeRoots": [
-      "./node_modules/@types"
+      "./node_modules/@types",
```

Comment:
Trailing commas make this non-standard JSON. It might work due to the specific TSC parser but I'm not entirely comfortable with it.